lcmgen/lcmgen.h

```diff
@@ -76,9 +76,13 @@ struct lcm_struct {
     char *lcmfile;  // file/path of function that declared it
     int64_t hash;
 
-    // Comments in the LCM type defition immediately before a struct is declared
+    // Comments in the LCM type defintion immediately before a struct is declared
```

Comment:
I think you may mean this?

```suggestion
    // Comments in the LCM type definition immediately before a struct is declared
```